packages/vats/test/bootstrapTests/bench-vaults-performance.js

```diff
@@ -22,8 +23,7 @@ import { makeWalletFactoryDriver } from './drivers.js';
 const test = anyTest;
 
 let snapshotNum = 0;
-const snapshotHeap = async step => {
-  console.log(`Snapshotting heap at step ${step}...`);
+const collectStats = async (step, dumpHeap) => {
```

Comment:
easy to assume that `step` is a number. also consider making `dumpHeap` optional.
```suggestion
/**
 * @param {string} step
 * @param {boolean} [dumpHeap]
 */
const collectStats = async (step, dumpHeap = false) => {
```